src/platform-implementation-js/dom-driver/gmail/views/gmail-thread-view.ts

```diff
@@ -243,6 +247,30 @@ class GmailThreadView {
       version,
     });
 
+    // AI suggestions container could be rendered after the subject container so
+    // if present, we need to adjust spacing when inserting the notice bar in between subject and AI suggestions
+    for (const [currentVersion, selector] of Object.entries(
+      this._subjectAISuggestionsContainerSelectors,
+    )) {
+      const hasAIButtonsSubjectContainer =
+        !!this._element.querySelector(selector);
+
+      if (!hasAIButtonsSubjectContainer) {
+        continue;
+      }
+
+      this._driver
+        .getLogger()
+        .eventSdkPassive('addNoticeBar AIButtonsSubjectContainer', {
```

Comment:
do you intend to break out the event name here?
```suggestion
        .eventSdkPassive('addNoticeBarAIButtonsSubjectContainer', {
```

Reply:
Good question! I was following the code style above (line 246):
<img width="727" alt="Screenshot 2024-04-26 at 2 05 19 PM" src="https://github.com/InboxSDK/InboxSDK/assets/108280206/a7d2e982-a13d-4194-b393-a1ad87b986a8">

I could keep it as is for consistency or swap both but the analytics reporting might break if we use it somewhere.

Reply:
gotcha, no need to change, i was just curious if you were following the pattern anywhere.